lib/jsanalyze.js

```diff
@@ -26,6 +26,23 @@ const path = require('path');
 const SOURCE_MAPPING_URL_REGEXP = /\/\/#[ \t]+sourceMappingURL=([^\s'"`]+?)[ \t]*$/mg;
 const __ = appc.i18n(__dirname).__;
 
+function sortObject (o) {
+	var sorted = {},
+		key,
+		a = [];
+
+	for (key in o) {
```

Comment:
In general, I think this function can be simplified to something like:

```
const sorted = {};
for (const key of Object.keys(o).sort()) {
    sorted[key] = o[key];
}
return sorted;
```

Reply:
:+1: fixed